Jenkinsfile

```diff
@@ -78,8 +78,11 @@ pipeline {
         stage('Azure Authenticator preparation - Allocate Azure Authenticator Instance') {
           steps {
             script {
-              node('azure-authn') {
-                env.AZURE_AUTHN_INSTANCE_IP = sh(script: 'curl icanhazip.com', returnStdout: true)
+              node('azure-linux') {
+                // get `ci/authn-azure/get_system_assigned_identity.sh` from scm
+                checkout scm
```

Comment:
`checkout scm` allows whatever is below it to run inside the Azure VM?

Reply:
it gets the whole source code, including the script that we need to run in the azure vm